plans/packit-integration.fmf

```diff
@@ -0,0 +1,20 @@
+discover:
+    how: fmf
+    url: https://github.com/packit/packit
+    filter: tier:0 | tier:1
```

Comment:
wow, this is so impressive; with 4 lines you get to run reverse dependency tests